Modules/mathmodule.c

```diff
@@ -2266,11 +2284,11 @@ math_log(PyObject *module, PyObject * const *args, Py_ssize_t nargs)
     if (!_PyArg_CheckPositional("log", nargs, 1, 2))
         return NULL;
 
-    num = loghelper(args[0], m_log);
+    num = loghelper(args[0], m_log, "math.log() expects a positive input, got '%R'.");
```

Comment:
I don't think using quotes around numbers is relevant. For instance, if it's a custom class of a float that overrides `__repr__`, then you don't necessarily want to surround the `__repr__` by quotes.

```suggestion
    num = loghelper(args[0], m_log, "math.log() expects a positive input, got %R.");
```